### PR TITLE
feat!: Add suspended type 

### PR DESCRIPTION
**Pull request by razbroc**
add status `supended` for jobs of Polygon Parts who failed multiple times and need manual attention.

| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✔                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                      |

Related issues: #MAPCO-5527

### CODE REVIEW

File: src/DAL/migration/v.2.6.2.sql [hunk 1]
```diff
@@ -26,4 +26,4 @@ ALTER TABLE "Job"
 DROP CONSTRAINT IF EXISTS "UQ_uniqueness_on_active_tasks_no_additional_identifier";
 
 ALTER TABLE "Job"
-ADD CONSTRAINT "UQ_uniqueness_on_active_tasks_no_additional_identifier" EXCLUDE ("resourceId" with =, "version" with =, "type" with =) WHERE ((status = 'Pending' OR status = 'In-Progress') AND ("additionalIdentifiers" IS NULL));
+ADD CONSTRAINT "UQ_uniqueness_on_active_tasks_no_additional_identifier" EXCLUDE ("resourceId" with =, "version" with =, "type" with =) WHERE ((status = 'Pending' OR status = 'In-Progress') AND ("additionalIdentifiers" IS NULL));
```
Comment (CL-SHLOMIKONCHA) [inline src/DAL/migration/v.2.6.2.sql]: revert - this file should not be change

File: src/DAL/migration/v2.6.3.sql
Comment (CL-SHLOMIKONCHA) [inline src/DAL/migration/v2.6.3.sql:1]: - v3.0.0 as it breaking changes
- add support for suspendedTasks to update_tasks_counters_insert

**Review by CL-SHLOMIKONCHA — CHANGES_REQUESTED** [review]

File: src/DAL/migration/v.2.6.2.sql (hunk 1 above)
Comment (razbroc) [reply src/DAL/migration/v.2.6.2.sql]: ok

File: src/DAL/migration/v2.6.3.sql
Comment (razbroc) [reply src/DAL/migration/v2.6.3.sql:1]: yessir

**Pull request closed without merging**